scripts/fetch_places_and_screens.exs

```diff
@@ -369,6 +369,9 @@ platform_to_stop_map =
   end)
   |> Enum.into(%{})
 
+{:ok, labels} = File.read("scripts/paess_labels.json")
```

Comment:
Just out of curiosity, do these custom labels change often? Or are changes pretty well communicated so we know to change them here when they change in Signs UI?